server/src/test/java/org/candlepin/dto/api/v1/EntitlementDTOTest.java

```diff
@@ -57,11 +57,11 @@ public EntitlementDTOTest() {
         pool.setProductName("pool_product_name");
 
         //TODO: change to ConsumerDTO once it gets introduced.
-        Consumer consumer = new Consumer();
+        ConsumerDTO consumer = new ConsumerDTO();
         consumer.setId("consumer_id");
         consumer.setName("consumer_name");
         consumer.setUsername("consumer_username");
```

Comment:
We need to set the uuid too here (e.g. `consumer.setUuid("consumer_uuid");`), since EntitlementDTO's toString/populate/equals/hashcode methods use the consumerDTO's uuid, and the tests for those methods for the consumer field are getting ignored.